packages/i18n/src/fi.ts

```diff
@@ -0,0 +1,83 @@
+import type { Language } from './types';
+
+// prettier-ignore
+const language: Language = {
+  code:               'fi',
+  schema:             (issue) => `Virheellinen tyyppi: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+  specific: {
+    base64:           (issue) => `Virheellinen Base64: Saatiin ${issue.received}`,
+    bic:              (issue) => `Virheellinen BIC: Saatiin ${issue.received}`,
+    bytes:            (issue) => `Virheelliset tavut: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    check:            (issue) => `Virheellinen syöte: Saatiin ${issue.received}`,
+    checkAsync:       (issue) => `Virheellinen syöte: Saatiin ${issue.received}`,
+    checkItems:       (issue) => `Virheellinen osio: Saatiin ${issue.received}`,
+    checkItemsAsync:  (issue) => `Virheellinen osio: Saatiin ${issue.received}`,
+    creditCard:       (issue) => `Virheellinen luottokortti: Saatiin ${issue.received}`,
+    cuid2:            (issue) => `Virheellinen Cuid2: Saatiin ${issue.received}`,
+    decimal:          (issue) => `Virheellinen desimaaliluku: Saatiin ${issue.received}`,
+    digits:           (issue) => `Virheellinen numeromäärä: Saatiin ${issue.received}`,
+    email:            (issue) => `Virheellinen sähköposti: Saatiin ${issue.received}`,
+    emoji:            (issue) => `Virheellinen emoji: Saatiin ${issue.received}`,
+    empty:            (issue) => `Virheellinen pituus: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    endsWith:         (issue) => `Virheellinen loppu: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    everyItem:        (issue) => `Virheellinen osio: Saatiin ${issue.received}`,
+    excludes:         (issue) => `Virheellinen sisältö: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    finite:           (issue) => `Virheellinen äärellinen luku: Saatiin ${issue.received}`,
+    graphemes:        (issue) => `Virheellinen grafeemimäärä: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    hash:             (issue) => `Virheellinen hash: Saatiin ${issue.received}`,
+    hexColor:         (issue) => `Virheellinen heksaväri: Saatiin ${issue.received}`,
+    hexadecimal:      (issue) => `Virheellinen heksadesimaali: Saatiin ${issue.received}`,
+    imei:             (issue) => `Virheellinen IMEI: Saatiin ${issue.received}`,
+    includes:         (issue) => `Virheellinen sisältö: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    integer:          (issue) => `Virheellinen kokonaisluku: Saatiin ${issue.received}`,
+    ip:               (issue) => `Virheellinen IP: Saatiin ${issue.received}`,
```

Comment:
I'd go with "IP-osoite", but OTOH, the original English is just "IP", so perhaps this is ok.